sshcode.go

```diff
@@ -608,3 +629,27 @@ func parseGCPSSHCmd(instance string) (ip, sshFlags string, err error) {
 
 	return strings.TrimSpace(userIP), sshFlags, nil
 }
+
+// When the user passes a path such as `/Workspace`, msys2 gives sshcode `C:/msys64/Worksapce`, which is not a valid remote path
+func gitbashWindowsDir(dir string) string {
+
+	// if dir is left empty, line82:main.go will set it to `~`, this makes it so that
+	// if dir is `~`, return `~/` instead of continuing with the gitbashWindowsDir()
+	// function. this prevens windows from trying to send a litteral instead of a relative
+	if dir == "~" {
+		return "~/"
+	}
+	mingwPrefix, _ := exec.Command("sh", "-c", "{ cd / && pwd -W; }").Output()
+	prefix := strings.TrimSuffix(string(mingwPrefix), "/\n")
+
+	// `pwd -W` returns a new line, not good so this removes
+	if strings.HasPrefix(dir, prefix) {
+		fmt.Println(prefix + dir)
+		resolved := strings.TrimPrefix(dir, prefix)
+		fmt.Println(resolved + dir)
+		flog.Info("Resolved windows path '%s' to '%s", dir, resolved)
+		return resolved
```

Comment:
```suggestion
		dir = strings.TrimPrefix(dir, prefix)
```

Reply:
```go
		fmt.Println(prefix + dir) // logging, will remove
		resolved := strings.TrimPrefix(dir, prefix) // agreed on lessening variables. 
		fmt.Println(resolved + dir) // logging, will remove
		flog.Info("Resolved windows path '%s' to '%s", dir, resolved) // This was (iirc) one of your additions, and could still be useful for debugging
		return resolved // still need to return a value
```

Reply:
```go
	if strings.HasPrefix(dir, prefix) {
		resolved := strings.TrimPrefix(dir, prefix)
		flog.Info("Resolved windows path '%s' to '%s", dir, resolved)
		return resolved
	}
```
it will `return dir` if there is nothing to do, (aka, a valid path is sent)